src/Admin/Ajax.php

```diff
@@ -21,23 +23,27 @@
  */
 class Ajax {
 
-    /**
-     * @var PayplugPermissions
-     */
-    private $permissions;
+	/**
+	 * @var PayplugPermissions
+	 */
+	private $permissions;
 
 	const REFRESH_KEY_ACTION = 'payplug_refresh_keys';
 	const CHECK_LIVE_PERMISSIONS = 'check_live_permissions';
 	const CHECK_BANCONTACT_PERMISSIONS = 'check_bancontact_permissions';
 	const CHECK_APPLEPAY_PERMISSIONS = 'check_applepay_permissions';
-	const CHECK_AMERICAN_EXPRESS_PERMISSIONS = 'check_american_express_permissions';
```

Comment:
why deleting american express checks?

Reply:
it was you that deleted this in this commit [96bff1b] :)